psiminer-core/src/main/kotlin/psi/method/MethodProvider.kt

```diff
@@ -19,4 +19,7 @@ abstract class MethodProvider {
     open fun stringsToCommentString(c: Collection<String>): String {
         return c.filterNot { it.none { s -> s.isLetterOrDigit() } }.flatMap { splitToSubtokens(it) }.joinToString("|")
     }
+
+    open class NotAMethodException(absentToken: String) :
+        IllegalArgumentException("Try to extract $absentToken not from the method")
```

Comment:
Let's change wording here. I suggest something like this: "Cannot extract $absentToken : class $className is not a method". For this, you should also pass the encountered class name.